TSQL_Parser/TSQL_Parser/Clauses/Parsers/TSQLOutputClauseParser.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+
+using TSQL.Tokens;
+
+namespace TSQL.Clauses.Parsers
+{
+	internal class TSQLOutputClauseParser
+	{
+		public TSQLOutputClause Parse(ITSQLTokenizer tokenizer)
+		{
+			TSQLOutputClause output = new TSQLOutputClause();
+
+			if (!tokenizer.Current.IsKeyword(TSQLKeywords.OUTPUT))
+			{
+				throw new InvalidOperationException("OUTPUT expected.");
+			}
+
+			output.Tokens.Add(tokenizer.Current);
+
+			while (
+				tokenizer.MoveNext() &&
+				!tokenizer.Current.IsCharacter(TSQLCharacters.Semicolon) &&
+				(
+					tokenizer.Current.IsKeyword(TSQLKeywords.AS) ||

```

Comment:
We should probably play out a lot of examples here. I think you need to add INTO, possibly as a sub clause?